src/profile-migrator/ephy-profile-migrator.c

```diff
@@ -698,6 +701,172 @@ migrate_app_desktop_file_categories (void)
 }
 
 static void
+parse_rdf_lang_tag (xmlNode  *child,
```

Comment:
Hm, I don't want to duplicate this code between ephy-profile-migrator and ephy-bookmarks-import. Since you already link it to libephymain.la you should just expose the extra API you need from ephy-bookmarks-import in its header file.

Which reminds me of one problem: how, in the new design, does the user import bookmarks from Firefox or Chrome? Is that TODO? Any idea where to put it when the existing bookmarks dialog is gone?